src/detect-util-uint.c

```diff
@@ -0,0 +1,233 @@
+/* Copyright (C) 2020 Open Information Security Foundation
```

Comment:
I think it'd be better to make the uint in the filename explit: u32. Also, I would like to see this as 'detect-engine'. So `detect-engine-u32.[ch]`

Reply:
> I think it'd be better to make the uint in the filename explit: u32

I thought that I would put the similar structures for uint16 in the same file as is done for `ByteExtractStringUint32` in util-byte.h

Do not you think so ?

> Also, I would like to see this as 'detect-engine'

Ok for renaming with engine